packages/stimulus/src/index.ts

```diff
@@ -3,9 +3,12 @@ import type { JSClient, JSSpan } from "@appsignal/types"
 export function installErrorHandler(appsignal: JSClient, application: any) {
   const prevHandler = application.handleError
 
-  application.handleError = function (error: Error, message: string) {
+  application.handleError = function(error: Error, message: string, detail: { identifier: string }) {
```

Comment:
Stimulus types simply define the [detail as `object`](https://github.com/hotwired/stimulus/blob/101a060f961e9240b55b29e25fdfe82c79b9c5b0/src/core/error_handler.ts#L2), so I didn't bother trying to pull in the real type here. Open to suggestions...

Reply:
I think not pulling in the real type is fine.

Out of an abundance of caution, we could account for `detail` potentially not including `identifier` by typing it as an optional attribute, and only setting the action if it is present. We do try our best to avoid being the reason why someone's application crashes in production.

That said, given that the code that invokes `handleError` has remained unchanged this long, and that there don't seem to be any circumstances in which a controller does not have an identifier, I'm not really concerned about it either way.

Reply:
> Out of an abundance of caution, we could account for `detail` potentially not including `identifier`

I'm going to actually go ahead and do this. Having looked at the code, I feel pretty confident it'll always be present, but it also seems reasonable for error handlers to be extra defensive, and this would also be some future-proofing insurance.